lms/lmsweb/tools/registration.py

```diff
@@ -129,6 +167,22 @@ def _build_user_text(user: UserToCreate) -> str:
         return msg
 
 
+def generate_confirmation_token(email: str) -> str:
+    return SERIALIZER.dumps(email, salt='email-confirmation')
+
+
+def send_confirmation_mail(email: str, fullname: str) -> None:
```

Comment:
I think we should have something like `Models/register.py`. Maybe even just `lmsweb/register.py`.
This function is too crucial to sit under `lms/lmsweb/tools/...`, it has a major role in the site functionality.

If you find another place to put it so it'll be organized it'll be great :)